druid/src/widget/container.rs

```diff
@@ -41,7 +41,7 @@ impl<T: Data> Container<T> {
         Self {
             background: None,
             border: None,
-            corner_radius: 0.0,
+            corner_radius: (0.0).into(),
```

Comment:
Mostly just out of curiosity, but is there a reason you went with the parentheses?

Reply:
Not really. I subconsciously thought it wouldn't parse without them, but it does. I'll remove them.